src/content/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results.mdx

```diff
@@ -9,6 +9,11 @@ redirects:
   - /docs/new-relic-only/new-relic-synthetics/using-monitors/viewing-monitor-results
   - /docs/synthetics/new-relic-synthetics/using-monitors/viewing-monitor-results
   - /docs/synthetics/new-relic-synthetics/using-monitors/view-monitor-results
+  - /docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results
+  - /docs/new-relic-only/new-relic-synthetics/using-monitors/collecting-synthetic-transaction-traces
+  - /docs/synthetics/new-relic-synthetics/using-monitors/collecting-synthetic-transaction-traces
+  - /docs/synthetics/new-relic-synthetics/using-monitors/collect-synthetic-transaction-traces
```

Comment:
```suggestion
  - /docs/synthetics/synthetic-monitoring/using-monitors/collect-synthetic-transaction-traces/
```

I think this is the link that needs to be there. I'm not sure what the other one leads to.

Reply:
Or maybe this in addition to the other links. In any case, this link needs to be the one to appear on this doc to redirect

Reply:
It looks like both links currently work so I think we can put both in pretty safely!